packages/amplify-gen1-codegen-function-adapter/API.md

```diff
@@ -0,0 +1,39 @@
+## API Report File for "@aws-amplify/amplify-gen1-codegen-auth-adapter"
```

Comment:
why is `auth-adapter` report under `function-adapter` ?

Reply:
Updated